filters/other/filter_42_DandelionSprouts_AdGuardHomeCompilationList _WebPushNotifications/configuration.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Dandelion Sprout's AdGuard Home Compilation List - Web Push Notifications",
```

Comment:
Can we make the name shorter? Dandelion Sprout's Web Push Notifications for instance

Reply:
In any case full names are visible in tooltips only
![image](https://user-images.githubusercontent.com/8361299/226305709-7ac25006-c394-4d90-b3e6-6481747fbb73.png)

Reply:
@DandelionSprout If you don't mind, we'll shorten the name.

Reply:
Maybe "Dandelion Sprout's Push Notifications Combined List", then, or something?

Reply:
@ameshkov

Reply:
@DandelionSprout Is `Dandelion Sprout's Web Push Notifications` ok? 19 symbols - only name.